### ISSUE TITLE
[BUG] getViewpointCoordinates() seemed not to work for bait on the negative strand

### ISSUE THREAD

**Issue by lipingtry** [Pasquali-lab/UMI4Cats#9]
## Context
Hi, When I used bait from the negative strand, getViewpointCoordinates() did not work. 

## Code

getViewpointCoordinates(
  bait_seq = "AAAGAGAATAAGAATAGTTATAATACAAA",
  bait_pad = "TTAGAGAT",
  res_enz = "GATC",
  ref_gen = BSgenome.Hsapiens.UCSC.hg19::BSgenome.Hsapiens.UCSC.hg19,
  sel_seqname = "chr11"
)

## prompt an error
Error in stop_if_wrong_length(what, ans_len) : 
  'ranges' must have the length of the object to construct (1) or length 1

## check the error trace
traceback()
5: stop(wmsg(what, " must have the length of the object ", "to construct (", 
       ans_len, ") or length 1"))
4: stop_if_wrong_length(what, ans_len)
3: new_GRanges("GRanges", seqnames = seqnames, ranges = ranges, 
       strand = strand, mcols = mcols, seqinfo = seqinfo)
2: GenomicRanges::GRanges(seqnames = sel_seqname, ranges = pos_viewpoint)
1: getViewpointCoordinates(bait_seq = "AAAGAGAATAAGAATAGTTATAATACAAA", 
       bait_pad = "TTAGAGAT", res_enz = "GATC", ref_gen = BSgenome.Hsapiens.UCSC.hg19::BSgenome.Hsapiens.UCSC.hg19, 
       sel_seqname = "chr11")

Do you know how to solve the problem? Thank you a lot.

**Comment by mireia-bioinfo** [comment]
Hi!

I've tried running the function and apparently the error is in giving `sel_seqname="chr11"`. If you run the function without this parameter it returns the correct result:

```
> getViewpointCoordinates(bait_seq="AAAGAGAATAAGAATAGTTATAATACAAA", 
    bait_pad="TTAGAGAT", 
    res_enz="GATC",  
    ref_gen=BSgenome.Hsapiens.UCSC.hg19::BSgenome.Hsapiens.UCSC.hg19)

GRanges object with 1 range and 0 metadata columns:
      seqnames        ranges strand
         <Rle>     <IRanges>  <Rle>
  [1]    chr11 355059-355099      -
  -------
  seqinfo: 1 sequence from an unspecified genome
```

We'll look into this bug, in the meantime if you remove the `sel_seqname` parameter everything should work fine.

**Comment by lipingtry** [comment]
Thank you very much！  @mireia-bioinfo

**Comment by mireia-bioinfo** [comment]
The fix has been merged with the master branch. Should be fixed from v 1.1.10.
